distribution/ddf-common/src/main/resources/data/countries.geo.json

```diff
@@ -12105,6 +12105,7 @@
       "type": "Feature",
       "id": "CHN",
       "properties": {
+        "geometryType": "country-shape",
```

Comment:
@GabrielFabian 	Where did this data originally come from? This might be an opportunity to contribute back to the source repo if China was the only country missing a `geometryType`

Reply:
China was the only one missing the country shape property for some reason. We add the country-shape property for our own data. The original repo doesn't have the country-shape property for any of the countries